remote_fs_test.go

```diff
@@ -56,7 +56,7 @@ func TestClone(t *testing.T) {
 	defer os.Remove(baseDir)
 
 	runTest(t, files, func(ctx context.Context, fs *remoteFS) {
-		err := fs.Clone(ctx, baseDir)
+		err := fs.Clone(ctx, baseDir, nil)
```

Comment:
Could you add a test case for the globbing to this suite?

Reply:
:+1: